packages/crashlytics/ios/Plugin/FirebaseCrashlyticsPlugin.swift

```diff
@@ -108,20 +111,39 @@ public class FirebaseCrashlyticsPlugin: CAPPlugin, CAPBridgedPlugin {
     }
 
     @objc func recordException(_ call: CAPPluginCall) {
-        guard let message = call.getString("message") else {
-            call.reject(errorMessageMissing)
-            return
+      guard let message = call.getString("message") else {
+        call.reject(errorMessageMissing)
+        return
+      }
+      
+      let stacktraceOptional = call.getArray("stacktrace")
+      let customPropertiesOptional = call.getArray("customProperties")
+        
+      guard !(stacktraceOptional != nil && customPropertiesOptional != nil) else {
+        call.reject(errorStacktraceAndCustomPropertiesDisallowed)
+        return
+      }
```

Comment:
Just use the stacktrace if both is passed and add a note to the method documentation that `customProperties` is ignored if `stacktrace` is passed.